main_training.py

```diff
@@ -55,9 +56,11 @@ def main(**kwargs):
 
     if cfg.low_cpu_fsdp:
         with torch.device("meta"):
-            model = LLaMA(llama_config, orig_init=True)
+            model = LLaMA(llama_config)
     else:
-        model = LLaMA(llama_config, orig_init=True)
+        model = LLaMA(llama_config)
+        
+    model = post_init_original(model)
```

Comment:
can we add a flag in the config like `use_original_llama_init`?  and default it to True

With this flag name, it also helps users to know what this line does.

Setting to False will use the "potentially superior" init that fms set, as it evolves in the future.

Reply:
is this a temporary workaround? I'm still not sure what the reasoning is for the current init scheme in the main repo, and suspect we're going to want to make "original" be the standard/default one.

Reply:
Sure, makes sense

Reply:
The init scheme in the current repo is more principled than the one here, which is extremely ad-hoc (a hardcoded constant initialization scale of 0.02). But in practice our formula doesn't drift too far from 0.02 for Llama 7b scale, and that init scheme does reflect the actual Llama2 model. So we probably should put 0.02 as the default init in openFMS, and put the more principled / auto-scaling one here as an optional setting